docs/walrus-sites/tutorial-install.md

```diff
@@ -8,30 +8,46 @@ prepare your environment for development.
 Before you start, make sure you
 
 - have a recent version of [Rust](https://www.rust-lang.org/tools/install) installed;
-- have `git` installed; and
 - followed all [Walrus setup instructions](../usage/setup.md).
 
 Then, follow these additional setup steps.
 
-## Clone the repository and build the `site-builder` tool
+## Installation
 
-First, clone and enter the Walrus Sites repo from <https://github.com/MystenLabs/walrus-sites>:
+Similar to the `walrus` client cli tool, we currently provide the `site-builder`
```

Comment:
Nit: the line length is wrong for most of the changes. It should be 100 chars, can you fix it?